<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="project_name">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Nome do Projeto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="project_name_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="project_desc">
       <property name="text">
        <string>Descrição</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="project_desc_edit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="task_name">
       <property name="placeholderText">
        <string>Adicionar nova tarefa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="task_add">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Colaboradores</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Alocar colaborador</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="colab_add"/>
          </item>
          <item>
           <widget class="QPushButton" name="add_colab">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Desalocar colaborador</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="colab_del"/>
          </item>
          <item>
           <widget class="QPushButton" name="del_colab">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>73</height>
        </size>
       </property>
       <property name="title">
        <string>Status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="pendente">
          <property name="text">
           <string>Pendente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="finalizado">
          <property name="text">
           <string>Finalizado</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>61</height>
        </size>
       </property>
       <property name="title">
        <string>Colaboradores Alocados</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="colab_qtd">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="task_list">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Lista de Tarefas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>499</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="excluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 239, 239);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>754</width>
        <height>519</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="painel_tasks"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>scrollArea</zorder>
  <zorder>task_list</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
